.devcontainer/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.154.2/containers/javascript-node/.devcontainer/base.Dockerfile
```

Comment:
I just tried to run on WSL/Docker, because I have a lot of issues with my computers.

Of course, it problematic because we are trying to launch a UI application.

So I think the Docker/WSL is working (don't remember which one) but with a very slow UI.